### PR TITLE
Rooms page

### PR DESCRIPTION
**Pull request by aboodswiti**
create rooms page with css

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^7.2.1",
     "axios": "^0.19.2",
     "react": "^16.13.1",
+    "react-axios": "^2.0.3",
```
Comment (yaqootturman) [inline client/package.json:11]: Why install react-axios?

File: client/src/App.js [hunk 2]
```diff
@@ -1,22 +1,28 @@
-import React from 'react'
+import React from "react";
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
+
+import Login from "./components/Layout/Login/Login.js";
```
Comment (yaqootturman) [inline client/src/App.js]: THis branch is for Rooms component ot Login?

File: client/src/components/Layout/Login/Login.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
```
Comment (yaqootturman) [inline client/src/components/Layout/Login/Login.js]: WHY!!!

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: client/src/components/Layout/rooms/Rooms.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  };
+
+  render() {
+    const { rooms } = this.state;
+    return (
+      <ul className="rooms">
+        {rooms.map((rooms) => {
+          return (
+            <div>
+              <div onClick={this.showInfo} className="rooms__card">
```
Comment (yaqootturman) [inline client/src/components/Layout/rooms/Rooms.js]: Okay, Its a good solution but not the best, avoid use CSS in react 
You can use prevState to toggle divs

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: client/src/components/Layout/rooms/Rooms.js (hunk 4 above)
Comment (aboodswiti) [reply client/src/components/Layout/rooms/Rooms.js]: Done

**Review by bayanseder — APPROVED** [review]

File: client/src/components/Layout/rooms/Rooms.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
```
Comment (yaqootturman) [inline client/src/components/Layout/rooms/Rooms.js]: Change the rooms folder name to Rooms

File: package.json [hunk 6]
```diff
@@ -28,6 +28,7 @@
     "express": "^4.17.1",
     "nodemon": "^2.0.3",
     "pg": "^8.0.2",
+    "react-axios": "^2.0.3",
```
Comment (yaqootturman) [inline package.json]: react-axios not used

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: client/src/components/Layout/rooms/Rooms.js (hunk 5 above)
Comment (aboodswiti) [reply client/src/components/Layout/rooms/Rooms.js]: Done

File: package.json (hunk 6 above)
Comment (aboodswiti) [reply package.json]: Done

**Review by yaqootturman — APPROVED** [review]

**Review by tasneembhiri — APPROVED** [review]

**Review by Hanan795 — APPROVED** [review]

**Pull request merged**